src/Compilers/VisualBasic/Portable/Syntax/InternalSyntax/SyntaxToken.vb

```diff
@@ -89,6 +89,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
                 Me._trailingTrivia = DirectCast(reader.ReadValue(), GreenNode)
             End Sub
 
+            Private ReadOnly Property IsReausable As Boolean Implements IObjectWritable.ShouldReuseInSerialization
```

Comment:
>IsReausable [](start = 38, length = 11)

```IObjectWritable.ShouldReuseInSerialization```?

Reply:
Sure

---
In reply to: [181834463](https://github.com/dotnet/roslyn/pull/26156#discussion_r181834463) [](ancestors = 181834463)